pmlc/dialect/stripe/into_mlir.cc

```diff
@@ -280,22 +308,63 @@ static void BlockIntoMLIR(OpBuilder* builder, const SymbolTable& outer, const st
     }
   }
 
+  // Process the block's execution location.
+  std::vector<mlir::Value*> executor;
+  if (!block.location.empty()) {
+    std::vector<TensorDim> dims;
+    for (std::size_t dev_idx = 0; dev_idx < block.location.devs.size(); ++dev_idx) {
+      const auto& dev = block.location.devs.at(dev_idx);
+      for (std::size_t unit_idx = 0; unit_idx < dev.units.size(); ++unit_idx) {
+        auto cls = (boost::format("%s_%zu_%zu") % dev.name % dev_idx % unit_idx).str();
```

Comment:
This seems to be defined in multiple places, is there a way to have the format of a 'cls' be defined in one spot, like a shared function or something?

Reply:
Done